gridstatus/nyiso.py

```diff
@@ -836,7 +839,8 @@ def _download_nyiso_archive(
 
             df = pd.read_csv(csv_url)
             df = _handle_time(df, dataset_name)
-            df["File Date"] = date.normalize()
+            if add_file_date:
```

Comment:
maybe since this only works when `dataset_name == 'isolf'`, you just check for that and always add in that case. no need for the `add_file_date` parameter

Reply:
Updated.